roles/kubernetes/node/tasks/main.yml

```diff
@@ -87,6 +87,7 @@
   changed_when: false
   check_mode: no
   register: sysctl_bridge_nf_call_iptables
+  check_mode: false
```

Comment:
This needs fixed -- check mode is defined [two lines above ](https://github.com/powerhome/kubespray/pull/13/files#diff-d8573eb6ba49a64e94117e08832eabf1319b675724997469cf0bea039bb0a331R88)

Reply:
Remove `check_mode: false`